pages/mesosphere/dcos/cn/1.13/cli/command-reference/dcos-marathon/dcos-marathon-app-update/index.md

```diff
@@ -7,41 +7,37 @@ excerpt: 更新应用程序
 render: mustache
 model: /mesosphere/dcos/1.13/data.yml
 enterprise: false
-render: mustache
-model: /mesosphere/dcos/1.13/data.yml
 ---
 
-# 说明
+## 说明
```

Comment:
Is this correct, first time I'm seeing anything other than English (Korean, Chinese)?

Reply:
Sorry, this is from Colin - forgot to log out of my team account!